src/pages/Home/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react';
+import MovieDetail from '../../components/MovieDetail';
+import MovieList from '../../components/CardList';
+import { getFromStorage } from '../../services';
+
+function Home() {
+  const [movieInfo, setMovieInfo] = useState(getFromStorage('selectedMovie'));
```

Comment:
fetching the information as the initial value of the useState variable is not a good practice, would be good fetching inside the useEffect hook